web/profiles/custom/yalesites_profile/config/sync/views.view.search.yml

```diff
@@ -0,0 +1,265 @@
+uuid: b4f36218-e053-4754-b5d0-0fccf46965c7
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.node.field_teaser_text
+    - search_api.index.node_index
+  module:
+    - search_api
+    - text
+    - user
+id: search
+label: Search
+module: views
+description: ''
```

Comment:
@marcb4k One small change requested. Can we add a description to the view since all of the other ones have a description? It can be something simple like: "Site search results page".